tests/setup.sh

```diff
@@ -44,6 +44,17 @@ if [[ "${OPENSSL_FORCE_FIPS_MODE}" = "1" || "$(cat /proc/sys/crypto/fips_enabled
     # We also need additional configuration in openssl.cnf to assume the token
     # is FIPS token
     TOKENOPTIONS="pkcs11-module-assume-fips = true"
+
+    # Force OpenSSL FIPS mode
+    export OPENSSL_FORCE_FIPS_MODE=1
+
+    # Force NSS softokn FIPS mode
+    export NSS_FIPS=1
```

Comment:
this needs to go to testvars otherwise it will be only in the setup environment, but not in the environment of the tests.